include/api/types.h

```diff
@@ -125,14 +125,14 @@ extern struct debug_syscall_error current_debug_error;
  * kernel is not performing their requested operation.
  */
 #define userError(M, ...) \
-    do {                                                                     \
-        out_error(ANSI_DARK "<<" ANSI_GREEN "seL4(CPU %lu)" ANSI_DARK        \
-                " [%s/%d T%p \"%s\" @%lx]: " M ">>" ANSI_RESET "\n",         \
-                SMP_TERNARY(getCurrentCPUIndex(), 0lu),                      \
-                __func__, __LINE__, NODE_STATE(ksCurThread),                 \
-                THREAD_NAME,                                                 \
-                (word_t)getRestartPC(NODE_STATE(ksCurThread)),               \
-                ## __VA_ARGS__);                                             \
+    do {                                                                       \
+        out_error(ANSI_DARK "<<" ANSI_GREEN "seL4(CPU %" SEL4_PRIu_word ")"    \
+                ANSI_DARK " [%s/%d T%p \"%s\" @%lx]: " M ">>" ANSI_RESET "\n", \
+                CURRENT_CPU_INDEX(),                                           \
```

Comment:
nitpick, are these light green squares spaces?

Reply:
They're just showing the diff from the removal of the SMP_TERNARY line.  Because there's a trailing \ at the end of the line, there needs to be new spaces added to make up for the removed non-space characters.